src/components/DailyScheduler.js

```diff
@@ -0,0 +1,220 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import DatePicker from "react-datepicker";
+import "../assets/stylesheets/timepicker-custom.css";
+import { add, isSameDay, isEqual, parseISO } from "date-fns";
+import ko from "date-fns/locale/ko";
+import styled from "styled-components";
+
+import Schedule from "./Schedule";
+import Modal from "./common/Modal";
+import { updateAvailableDates } from "../features/counselorSlice";
+import {
+  TYPE_DATE,
+  EXISTED_TIMELINE_WARNING,
+  START_DATE,
+  END_DATE,
+} from "../constants/date";
+
```

Comment:
```suggestion
import React, { useState } from "react";
import { useSelector, useDispatch } from "react-redux";
import DatePicker from "react-datepicker";
import { add, isSameDay, isEqual, parseISO } from "date-fns";
import ko from "date-fns/locale/ko";
import styled from "styled-components";

import Schedule from "./Schedule";
import Modal from "./common/Modal";
import { updateAvailableDates } from "../features/counselorSlice";
import {
  TYPE_DATE,
  EXISTED_TIMELINE_WARNING,
  START_DATE,
  END_DATE,
} from "../constants/date";

import "../assets/stylesheets/timepicker-custom.css";
```
CSS파일은 패키지와 따로 분리하여야 될 것 같아요!